packages/swing-store/src/bundleStore.js

```diff
@@ -229,10 +238,9 @@ INSERT INTO bundles (bundleID, bundle) VALUES (?, NULL)
     // prettier-ignore
     (parts.length === 2 && type === 'bundle') ||
       Fail`expected artifact name of the form 'bundle.{bundleID}', saw ${q(name)}`;
-    const row = sqlGetBundle.get(bundleID);
-    row || Fail`bundle ${q(bundleID)} not found`;
-    const rawBundle = row.bundle;
-    rawBundle || Fail`bundle ${q(bundleID)} pruned`;
+    const row =
+      sqlGetBundle.get(bundleID) || Fail`bundle ${q(bundleID)} not found`;
+    const rawBundle = row.bundle || Fail`bundle ${q(bundleID)} pruned`;
```

Comment:
So far I've found that I don't care for `good_condition || Fail..`, because it makes me have to think about what's being short-circuited (it's not immediately obvious that the left-hand expression is a good thing, rather than a bad thing). But this form doesn't bug me in that same way, maybe because it reads more like english, or maybe the fact that it needs to evaluate to something makes it more clear that the bad_condition would be if the left expression were falsy. Eh.